src/commands/wizard/onboardingWizard.ts

```diff
@@ -33,6 +33,10 @@ async function runPostProjectConfigurationSteps(
         await AuthorizeCommand.authorizeToOrg();
         await DeployToOrgCommand.deployToOrg();
 
+        await new LwcGenerationCommand(
```

Comment:
Are we ready to enable this functionality, after this PR goes in? If not, we should not put this in the PR.

Reply:
After this PR goes in the remaining thing will be to get the compact layout. We can disable the functionality until that is in.

Reply:
Removed it. Will continue to put the command on dev's machine locally until we're ready to enable the functionality. cc: @dbreese